Pocket Casts/dist/metadata.json

```diff
@@ -0,0 +1,20 @@
+{
+  "author": {
+    "name": "ry",
+    "id": "421371986824921109"
+  },
+  "service": "Pocket Casts",
+  "description": "All the features you need in a podcasting app without any of the bloat. Podcast listening turned up to 11.",

```

Comment:
You are missing the language identifier
"description": {
    "en": "DESCRIPTION"
 },

Reply:
bro it's not necessary